controllers/configmap_controller.go

```diff
@@ -579,8 +580,9 @@ func (r *ConfigMapReconciler) ensureWICDRoleBinding() error {
 	}
 	if err == nil {
 		// check if existing RoleBinding's contents are as expected, delete it if not
-		if existingRB.RoleRef == expectedRB.RoleRef &&
-			len(existingRB.Subjects) == len(expectedRB.Subjects) && existingRB.Subjects[0] == expectedRB.Subjects[0] {
+		if reflect.DeepEqual(existingRB.RoleRef, expectedRB.RoleRef) &&
```

Comment:
Using deepequal for RoleRef seems uncessesary here. You only need to check that `RoleRef.Name` is equivalent. APIGroup and kind will never be different.